purchase_blanket_order/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019-22 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Blanket Orders",
+    "category": "Purchase",
+    "license": "AGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "version": "17.0.1.0.0",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "summary": "Purchase Blanket Orders",
+    "depends": [
+        "purchase",
+        "web_action_conditionable",
```

Comment:
Looking at the module, this dependency looks like it is not needed. Can you check if it is really the case? If not, we probably can get rid of it during this migration.

Reply:
You are right, I tested it and it looks like the module no longer needs this dependency. Removing it.